src/index.html

```diff
@@ -1,22 +1,25 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <meta name="format-detection" content="telephone=no">
-    <meta name="msapplication-tap-highlight" content="no">
-    <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width">
-    <meta name="mobile-web-app-capable" content="yes">
-    <meta name="apple-mobile-web-app-capable" content="yes">
-    <meta name="apple-mobile-web-app-status-bar-style" content="black">
-    <meta name="description" content="karrot">
+    <meta charset="utf-8" />
+    <meta name="format-detection" content="telephone=no" />
+    <meta name="msapplication-tap-highlight" content="no" />
+    <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width" />
+    <meta name="mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black" />
     <meta name="theme-color" content="#4a3520" />
+    <meta name="description" content="Modern website for organization of foodsaving groups worldwide" />
+    <meta property="og:title" content="Karrot" />
+    <meta property="og:image" content="statics/carrot_logo.png" />
+    <meta property="og:description" content="Modern website for organization of foodsaving groups worldwide" />
 
     <title>(karrot loading...)</title>
-    <link rel="shortcut icon" href="statics/carrot_logo.ico">
-    <link rel="icon" type="image/png" href="statics/carrot_logo.png" sizes="32x32">
+    <link rel="shortcut icon" href="statics/carrot_logo.ico" />
+    <link rel="icon" type="image/png" href="statics/carrot_logo.png" sizes="32x32" />
   </head>
   <body>
-    <div id="q-app"></div>
+    <div id="q-app" />
```

Comment:
This is _real_ html here, and self-closing `<div>` is invalid. We can use it in the vue templates as it's loaded by a more relaxed parser of our choosing, this file is parsed by the parser of the unknown users unknown browsers choosing.

[html5 spec says](https://www.w3.org/TR/html5/syntax.html#start-tags) (about opening tags):

> Then, if the element is one of the void elements, or if the element is a foreign element, then there may be a single "/" (U+002F) character. This character has no effect on void elements, but on foreign elements it marks the start tag as self-closing.

- void elements _area, base, br, col, embed, hr, img, input, link, meta, param, source, track, wbr_
- foreign elements are _Elements from the MathML namespace and the SVG namespace._

Reply:
Ah good! Didn't know that, will fix/undo these changes.